src/main/java/uk/gov/hmcts/reform/sscs/ccd/domain/SscsDocument.java

```diff
@@ -24,10 +24,14 @@ public SscsDocument(@JsonProperty("value") SscsDocumentDetails value) {
     public int compareTo(SscsDocument doc2) {
         SscsDocumentDetails nextDocumentDetails = doc2.getValue();
 
-        if (value.getDocumentDateAdded() == null) {
+        if (value.getDocumentDateAdded() == null || doc2.getValue().getDocumentDateAdded() == null) {
```

Comment:
Won't this create inconsistent sorting? if the doc date is null, it goes below doc2, but if the doc has a date and the doc2 date is null then the doc is put below doc2 as well.

Reply:
Yeh, good point. I'll fix